en_ID/pages/profiles/FatwaDwiHidayat.md

```diff
@@ -0,0 +1,3 @@
+#FatwaDwiHidayat
+
+Hi I'm [FatwaDwiHidayat](http://github.com/FatwaDwiHidayat). I like software engineering because it is fun! And my favorite food is burger.
```

Comment:
This is too simple, try to be more creative ya

Reply:
You can put images, videos, or any additional things that describe you!

Reply:
I had doing my revision. Please review ya